docs/index.md

```diff
@@ -4,7 +4,7 @@ description: Brief overview of Pi-hole
 last_updated: Thur May 02 01:00:00 2019 UTC
 ---
 <p align="center">
-<a href="https://pi-hole.net"><img src="https://pi-hole.github.io/graphics/Vortex/Vortex_with_text.png" width="150" height="255" alt="Pi-hole"></a><br/>
+<a href="https://pi-hole.net"><img src="images/Vortex_with_text.png" width="150" height="255" alt="Pi-hole"></a><br/>
```

Comment:
@dschaper is this ok with you? I remember there was some initiative before to centralize the images used on various readmes and documentation sites.

Reply:
If you want to include the graphics repo as a submodule to this repo and then pull directly, that would work.